crates/control/src/search_suggestor.rs

```diff
@@ -195,11 +194,9 @@ fn get_rule_hypotheses(
     }
 }
 
-fn conservative_kicking_team_half(kicking_team: Team) -> Half {
+fn kicking_team_half(kicking_team: Team) -> Half {
```

Comment:
What do you think of adding an into trait impl for this?

Reply:
I'm not so sure about this. The mapping between team hand half may be flipped depending on the logic one is trying to implement. I'd rather have well named functions for this than implement the into trait.